cts/valgrind-pcmk.suppressions

```diff
@@ -160,20 +161,45 @@
 }
 
 {
-   glib mainloop internals - run
+   glib - mainloop run malloc 3
    Memcheck:Leak
    fun:malloc
    fun:g_malloc
    fun:g_slice_alloc
-   fun:g_slice_alloc0
-   obj:*/libglib-2.*
+   ...
    fun:g_main_context_dispatch
-   obj:*/libglib-2.*
+   ...
```

Comment:
Avoid `...` after `g_main_loop_run()`. That catches all daemon code paths. (Valgrind isn't detecting that the leak originated in `g_main_loop_run()`, only that it was in the call stack, and it is at the bottom of all code paths when the main loop is running.)